site/content/docs/manifest/lb-web-service.md

```diff
@@ -71,10 +71,13 @@ image:
   build:
     dockerfile: path/to/dockerfile
     context: context/dir
+    target: build-stage
+    cache_from:
+      - image:tag
     args:
       key: value
 ```
-In this case, copilot will use the context directory you specified and convert the key-value pairs under args to --build-arg overrides. The equivalent docker build call will be: `$ docker build --file path/to/dockerfile --build-arg key=value context/dir`.
+In this case, copilot will use the context directory you specified and convert the key-value pairs under args to --build-arg overrides. The equivalent docker build call will be: `$ docker build --file path/to/dockerfile --target build-stage --cache-from image:tag --build-arg key=value context/dir`.
```

Comment:
Would you mind adding this blurb to the new `manifest/scheduled-job.md` as well?